helm/fullnode/values.yaml

```diff
@@ -0,0 +1,123 @@
+chain:
+  era: 24
+  name: testnet
+  genesisConfigmap:
+
+diem_chains:
+  testnet:
+    id: TESTNET
+    waypoint: 0:3139c30efe6dbde4228efb9df32c137dc3a2490b97ab6a086897be1d0cb336f0
+    seed_addrs:
+      4223dd0eeb0b0d78720a8990700955e1:
+      - "/dns4/fn.testnet.diem.com/tcp/6182/ln-noise-ik/d29d01bed8ab6c30921b327823f7e92c63f8371456fb110256e8c0e8911f4938/ln-handshake/0"
+
+rust_log: info
+
+image:
+  repo: diem/validator
+  tag: release-1.1_a1abfc34
```

Comment:
how do we want to manage this version? Do we want to set it to `devnet`, or should we pass it in as  a value?

Reply:
Probably best to set `testnet`, but I don't think we're tagging our images that way currently. That's just the latest release. We can write some docs for how to change the `image.tag` though

Reply:
there is a `devnet` tag on `libra` images that's coming to `diem` images soon too. 

For this though it might make sense to have it configurable via values, since we want all of the versions to be the same across images, that would make it easier (we can have configurable overrides per, as well)

Reply:
Yea makes sense. I've added some instructions in the README to be a bit more clear.